rules/windows/command_and_control_dns_tunneling_nslookup.toml

```diff
@@ -14,6 +14,42 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "kuery"
 license = "Elastic License v2"
 name = "Potential DNS Tunneling via NsLookup"
+note = """## Triage and analysis
+
+### Investigating Potential DNS Tunneling via NsLookup
+
+Attackers can abuse existing network rules that allow DNS communication with external resources to use the protocol as
+their command and control and/or Exfiltration channel.
+
+DNS queries can be used to infiltrate data like commands to be run, malicious files, etc., and also for exfiltration,
+since queries can be used to send data to the attacker-controlled DNS server. This process is commonly known as DNS Tunneling.
```

Comment:
```suggestion
DNS queries can be used to infiltrate data such as commands to be run, malicious files, etc., and also for exfiltration,
since queries can be used to send data to the attacker-controlled DNS server. This process is commonly known as DNS tunneling.
```